docs/tools/flow-cli/migrate/migrate-emulator-state.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Migrating Emulator state to Cadence 1.0
+description: How to locally test storage migration and staged contract upgrades on Emulator
+sidebar_position: 6
+---
+
+Flow CLI provides a command `flow migrate state` to migrate your local emulator
+state to Cadence 1.0, and to deploy upgraded contracts to the emulator.
+This is useful for locally testing the staged contract upgrades, and to see how the
+new network and the upgraded contracts would behave with the migrated data.
+
+## Migration Guide
+
+To test the migration, first you would need an emulator state, created with an emulator/CLI release
+that uses a pre-1.0 Cadence version.
+
+### Getting the old state
+
+For example, to get an emulator state with Flow CLI `1.14.0`:
+
+- Start the emulator with the `--persist` flag.
+  ```shell
+    flow emulator --persist
+  ```
+- Deploy the pre-1.0 project/contracts to the emulator
+- Run transactions if there are any.
+- Close the emulator. It is important to make sure the emulator is closed/shutdown before taking the snapshot,
```

Comment:
What does "close" mean here? Stopping? Maybe provide an example, like Ctrl-C on *nix platforms, and Ctrl-Break (or Ctrl-Pause) on Windows

Reply:
Yes, Improved the wording